resources/aws_ec2.go

```diff
@@ -394,6 +394,9 @@ func (obj *AwsEc2Res) longpollWatch() error {
 	}()
 	obj.wg.Add(1)
 	go func() {
+		// shorten some errors we need throughout
+		errCancelled := request.CanceledErrorCode
```

Comment:
amazingly, this is actually a constant! https://godoc.org/github.com/aws/aws-sdk-go/aws/request#pkg-constants

let's not re-define it, let's just use it directly.